tracing/src/main/java/com/palantir/tracing/Tracers.java

```diff
@@ -37,6 +38,9 @@
      */
     public static final String TRACE_SAMPLED_KEY = "_sampled";
 
+    /** The key under which tracing request ids are inserted into SLF4J {@link org.slf4j.MDC MDCs}. */
+    public static final String REQUEST_ID_KEY = "_requestId";
```

Comment:
This will appear in every log line. Do we want to abbreviate it in some way? As written it's self-explanatory.

Reply:
I like it as is